### PR TITLE
update(react): Migrate from the deprecated React.propTypes to prop-types package

### PR DESCRIPTION
**Pull request by gucheen**
Migrate from the deprecated React.propTypes to prop-types package

https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "jsdom": "^9.0.0",
     "mocha": "^2.3.4",
     "mocha-eslint": "^2.0.2",
+    "prop-types": "^15.5.6",
```
Comment (ljharb) [inline package.json]: Must be a dependency, not a dev dependency.

**Comment by gucheen** [comment]
@ljharb It has been changed. Thanks for the suggestions.

**Comment by tajo** [comment]
Thank you!

**Comment by tajo** [comment]
Published as `v3.1.0`.

**Pull request merged**